pkg/component/operator/web/v0/config/tasks.json

```diff
@@ -108,101 +108,37 @@
-                }
-              },
-              "required": [
-                "filter-method"
+          "properties": {
+            "exclude-patterns": {
+              "description": "When the URL is matched, the URL will not be crawled.",
+              "instillAcceptFormats": [
+                "array:string"
               ],
-              "title": "Regex",
-              "type": "object"
-            },
-            {
-              "properties": {
-                "filter-method": {
-                  "const": "Substring",
-                  "type": "string",
-                  "title": "Filter Method",
-                  "description": "Filtering based on substring. Regex is more powerful than Substring. However, Substring is easier to use.",
-                  "instillUIOrder": 0
-                },
-                "exclude-substrings": {
-                  "description": "When the URL contains the substrings, the URL will not be crawled.",
-                  "instillAcceptFormats": [
-                    "array:string"
-                  ],
-                  "items": {
-                    "type": "string"
-                  },
-                  "instillUIOrder": 1,
-                  "type": "array",
-                  "title": "Exclude Substring"
-                },
-                "include-substrings": {
-                  "description": "When the URL contains the substrings, the URL will be crawled.",
-                  "instillAcceptFormats": [
-                    "array:string"
-                  ],
-                  "items": {
-                    "type": "string"
-                  },
-                  "instillUIOrder": 2,
-                  "type": "array",
-                  "title": "Include Substring"
-                },
-                "case-sensitive": {
-                  "description": "A flag indicating whether the substring matching is case-sensitive. When it is true, the matching is case-sensitive. When it is false, the matching is case-insensitive. The default value is false. For example, when it is case-sensitive, cat would only match 'cat' but not 'Cat' or 'CAT'. When cat is case-insensitive, on the other hand, would match 'cat', 'Cat', 'CAT', or any other variation of uppercase and lowercase letters.",
-                  "instillAcceptFormats": [
-                    "boolean"
-                  ],
-                  "instillUIOrder": 3,
-                  "title": "Case Sensitive",
-                  "type": "boolean"
-                }
+              "items": {
+                "type": "string"
               },
-              "required": [
-                "filter-method"
+              "instillUIOrder": 1,
+              "type": "array",
+              "title": "Exclude Patterns"
+            },
+            "include-patterns": {
+              "description": "When the URL is matched, the URL will be crawled.",
+              "instillAcceptFormats": [
+                "array:string"
```

Comment:
If regexp has an OR filter (as  in the examples you provided), what's the use case for passing several expressions?